hx_requests/hx_requests.py

```diff
@@ -153,6 +150,11 @@ def get_headers(self, **kwargs) -> Dict:
                 headers["HX-Redirect"] = self.redirect
         if self.no_swap:
             headers["HX-Reswap"] = "none"
+
+        if hasattr(self, "get_triggers"):
+            triggers = self.get_triggers(**kwargs)
```

Comment:
I like it, but can you explicitly make the function `get_triggers` on the base hx request. I think it would be clearer and then you can also type it to return a list so that its clear that the function needs to return a list of headers.

Reply:
Updated.